spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/utils/RewriteRowLevelOperationHelper.scala

```diff
@@ -108,6 +133,23 @@ trait RewriteRowLevelOperationHelper extends PredicateHelper with Logging {
     Project(Seq(findOutputAttr(agg.output, FILE_NAME_COL)), agg)
   }
 
+  private def buildPlanWithFileAccumulator( fileAccum: SetAccumulator[String],
+                                            prunedTargetPlan: LogicalPlan): LogicalPlan = {
+    val fileAttr = findOutputAttr(prunedTargetPlan.output, FILE_NAME_COL)
+    val rowPosAttr = findOutputAttr(prunedTargetPlan.output, ROW_POS_COL)
+    val accumulatorExpr = Alias(AccumulateFiles(fileAccum, fileAttr), AFFECTED_FILES_ACC_ALIAS_NAME )()
```

Comment:
Style nit: the space between `AFFECTED_FILES_ACC_ALIAS_NAME` and `)` is unnecessary.